### ISSUE TITLE
Style/TrailingCommaInArguments false positive with interpretation of multi-lined arguments?

### ISSUE THREAD

**Issue by brandonweiss** [rubocop-hq/rubocop#3586]
If I set this configuration:

``` yaml
Style/TrailingCommaInArguments:
  Enabled: true,
  EnforcedStyleForMultiline: consistent_comma
```

Then it enforces this code should have a comma like this:

``` ruby
EmailWorker.perform_async({
  subject: "hey there",
  email: "foo@bar.com"
},)
```

This doesn't seem right to me? It's a nuanced difference, but I don't consider this multi-lined arguments—the argument actually starts on the same line as the parentheses. The argument itself is multi-lined for readability, but the arguments aren't multi-lined. This is what I think multi-lined arguments look like:

``` ruby
EmailWorker.perform_async(
  "foo",
  "bar,
)
```

Or in the context of a hash:

``` ruby
EmailWorker.perform_async(
  {
    subject: "hey there",
    email: "foo@bar.com"
  },
)
```

Thoughts?

**Comment by jonas054** [comment]
I agree. The purpose of the trailing comma cops is to make it possible to enforce a style that minimizes the diff when another element is added at the end. And the comma in your example is useless for that purpose.

In this code from `lib/rubocop/cop/mixin/trailing_comma.rb`, we see that the intentions are correct, but the implementation is wrong.

```ruby
      # Returns true if the round/square/curly brackets of the given node are
      # on different lines, and each item within is on its own line, and the
      # closing bracket is on its own line.
      def multiline?(node)
```

The closing bracket, `)` in your case, is *not* on its own line.

I noted BTW that you also need the following configuration to reproduce the problem:
```yaml
Style/BracesAroundHashParameters:
  EnforcedStyle: braces
```

**Comment by brandonweiss** [comment]
@jonas054 ❤️
